generate-all.sh

```diff
@@ -1,10 +1,12 @@
 #!/bin/bash
 
+GOOGLE_LOCATION="us-central1-c"
```

Comment:
That could be set as a repo env or secret right (I'd prefer the former since it's nothing really mission critical where we keep our dev clusters which are used mostly for CI and testing only) ? And then the script could get `set -u` to error in unassigned variables

Reply:
Yeah, why not 👍 PTAL